plctestbench/node.py

```diff
@@ -66,9 +66,13 @@ def _save_to_database(self):
         self._get_database().add_node(entry, type(self).__name__)
 
     def get_id(self) -> str:
-        return str(hash(self.settings))
+        hash_properties = Settings.__copy__(self.settings)
+        hash_properties.add("parent", self.parent.get_id() if self.parent!=None else "")
```

Comment:
Perché aggiungi l'hash del parent qui? Non dovrebbe essere più necessario in seguito alle modifiche apportate con questo commit: 67bc636a187a8f374cbb16593eb360c01c021e1a

Reply:
La modifica era precedente alle tue fix, ma per evitare problemi su una cosa così delicata non l'avevo ancora rimossa. Se è ridondante alla luce delle tue fix ignorala pure.